library/azure_rm_networkinterface.py

```diff
@@ -158,6 +158,16 @@
                     - Whether the ip configuration is the primary one in the list.
                 type: bool
                 default: 'no'
+            application_security_groups:
+                description:
+                    - List of application secuirty groups in which the IP configuration is included.
+                    - Element of the list could be a resource id of application security group, or dict of resource_group and name.
+                    suboptions:
```

Comment:
I am not quite sure whether we should define suboptions like this for "raw" type.
I think the description above, that it could be an resource id or dict containing resource_group and name is sufficient and there's no need to describe suboptions

Reply:
fixed